test/Microsoft.DotNet.ProjectJsonMigration.Tests/ProjectJsonBuilder.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Microsoft.DotNet.TestFramework;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.ProjectJsonMigration.Tests
+{
+    /// <summary>
+    /// Used to build up test scenario project.jsons without needing to add a new test asset.
+    /// </summary>
+    public class ProjectJsonBuilder
+    {
+        private static readonly string s_defaultProjectJsonTestAsset = "TestAppWithRuntimeOptions";
```

Comment:
Agreed, I think the default should probably just go away. 

The ProjectJsonMigration Unit Tests don't have access to the template, it's passed in via the cli with dotnet new